src/Fieldtypes/BaseFieldtype.php

```diff
@@ -155,6 +155,13 @@ protected function toItemArray($id)
         } else {
             $record = $id;
         }
+        
+        if (is_null($record)) {
+            return [
```

Comment:
Would you be able to return something like this instead? (as per the [Entry fieldtype](https://github.com/statamic/cms/blob/5b8d7724f45230598c4e7f010a2d32c0de309bbc/src/Fieldtypes/Relationship.php#L219))

```php
return [
    'id' => $id,
    'title' => $id,
    'invalid' => true,
];
```

I think passing in `invalid` means that the relationship field shows as red so the user can fix the relationship.